Examples/ImageGridManipulation/ImageGridManipulation.R

```diff
@@ -0,0 +1,67 @@
+#=========================================================================
+#
+#  Copyright Insight Software Consortium
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#=========================================================================
+# Run with:
+#
+# Rscript --vanilla ImageGridManipulation.R input-1 input-2
+#
+
+library(SimpleITK)
+
+args <- commandArgs( TRUE )
+
+if (length(args) <  2){
+   write("Usage arguments: <input-1> <input-2>", stderr())
+   quit(save="no", status=1)
+}
+
+# Two vector images of same pixel type and dimension expected
+reader <- ImageFileReader()
+reader$SetFileName(args[[1]])
+image_1 <- reader$Execute()
+reader$SetFileName(args[[2]])
+image_2 <- reader$Execute()
+
+# Join two N-D Vector image images to form an (N+1)-D image
```

Comment:
repetition "Vector image images", correct to "Vector images..." (also fix in the Python version)